docs/pages/docs/advanced/foundry.mdx

```diff
@@ -0,0 +1,126 @@
+---
+title: "Integrate with Foundry"
+description: "A guide for using Ponder with the Foundry smart contract development toolchain."
+---
+
+import { Callout, Steps } from "nextra/components";
+
+# Foundry
+
+This guide describes how to integrate Ponder and Foundry during local development.
+
+Foundry projects follow various development workflows (test-driven, deploy to a fresh chain, deploy to a fork, etc). Rather than a one-size-fits-all integration, this page offers patterns that you can adapt to your workflow.
+
+The Foundry [example project](https://github.com/ponder-sh/ponder/tree/main/examples/with-foundry) includes a simple implementation of each pattern described in this guide.
```

Comment:
I would delete this sentence. "pattern" isn't referring to the same thing that you mentioned in the paragraph above.